docs/habana-gaudi.md

```diff
@@ -142,11 +142,12 @@ The [`SFTTrainer`](https://huggingface.co/docs/trl/sft_trainer) from `trl` does
 
 ## Install and run InstructLab with Intel Gaudi
 
-Install `InstructLab` from checkout with additional dependency `habana`:
+Install `InstructLab` from checkout with additional dependencies:
 
 ```shell
 . $HABANALABS_VIRTUAL_DIR/bin/activate
```

Comment:
Since every mention of `habana` is gone, is `$HABANALABS_VIRTUAL_DIR` still needed? Or needs to be replaced?

Reply:
It's still required. The Habana Labs installer creates a custom virtual env.